.buildkite/pipeline.yml

```diff
@@ -217,3 +217,9 @@
             "--environment-id", "stage",
             "--description", $BUILDKITE_BUILD_URL,
             "--confirmation-wait-for", 3600]
+
+- wait
+
+- label: "Integration pipeline"
+  trigger: "integration"
+  async: true
```

Comment:
Do we want this async? The next step here could be an automated prod deployment if it were not.

Reply:
Going to keep it like this for now, but yes, indeed that is The Dream™.